features/makepot.feature

```diff
@@ -2458,3 +2458,391 @@ Feature: Generate a POT file of a WordPress project
       Success: POT file successfully generated!
       """
     And the contents of the result.pot file should match /^msgid/
+
+  Scenario: Extract strings from block.json files
+    Given an empty foo-plugin directory
+    And a foo-plugin/foo-plugin.php file:
+      """
+      <?php
+      /**
+       * Plugin Name: Foo Plugin
+       */
+      """
+    And a foo-plugin/block.json file:
+      """
+      {
+        "name": "my-plugin/notice",
+        "title": "Notice",
+        "category": "common",
+        "parent": [ "core/group" ],
+        "icon": "star",
+        "description": "Shows warning, error or success notices  ...",
+        "keywords": [ "alert", "message" ],
+        "textDomain": "foo-plugin",
+        "attributes": {
+          "message": {
+            "type": "string",
+            "source": "html",
+            "selector": ".message"
+          }
+        },
+        "styleVariations": [
```

Comment:
We also settled on `styles` to avoid aliases and respect the first choice. The reasoning behind it is as follows: even if it is confusing in the first place, it's also double as much confusing with an alias :)